specification/security/get_builtin_privileges/SecurityGetBuiltinPrivilegesRequest.ts

```diff
@@ -23,5 +23,6 @@ import { RequestBase } from '@_types/Base'
  * @rest_spec_name security.get_builtin_privileges
  * @since 7.3.0
  * @stability stable
+ * @cluster_privileges manage_enrich
```

Comment:
Forgive my ignorance: Not sure why `manage_enrich` is specifically needed here. Is this comment kinda a change history?

Reply:
Happy to answer every question! In the issue you wrote:

> The manage_enrich cluster privilege is missing from:
> Response of the [Get builtin privileges API](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-get-builtin-privileges.html)
> Request of the [Create/Update role API](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-put-role.html) where this privilege should be a candidate for value of the cluster field.

This change adds the `manage_enrich` to the two APIs. If you look at the `schema.json`, now the new privileges are listed there as well.

Reply:
@delvedor From my reading the enum value was only missing in the response itself, not as a `@cluster_privileges` annotation for this API so adding the value to the enum is sufficient. From the docs it seems that the privilege needed for this API is actually `manage_security`.

Reply:
Sorry for any confusion that may have been caused by my original issue. @sethmlarson is right. The cluster privilege requried for calling this API is `manage_security`. The response of this API should return a list of cluster privileges which should include the missing `manage_enrich` privilege. 

Btw: I am assuming the `@cluster_privileges` directive is to specify what cluster privilege is required for calling the API.